console/src/components/Services/Settings/MetadataOptions/MetadataOptions.js

```diff
@@ -95,8 +98,14 @@ const mapStateToProps = state => {
     dataHeaders: { ...state.tables.dataHeaders },
   };
 };
-
+const mapDispatchToProps = dispatch => {
+  return {
+    // dispatching plain actions
+    exportSchema: () => dispatch(exportSchema()),
```

Comment:
```suggestion
    exportSchema: () => dispatch(exportSchema),
```

and simplify `exportSchema`.